danceschool/vouchers/views.py

```diff
@@ -63,9 +63,10 @@ def form_valid(self,form):
         logger.info('Processing gift certificate.')
 
         try:
+            currency_symbol = getConstant('general__currencySymbol')
             voucher = Voucher.create_new_code(
                 prefix='GC_',
-                name=_('Gift certificate: %s%s for %s' % (getConstant('general__currencySymbol'),self.amount, emailTo)),
+                name=_('Gift certificate: %s%s for %s' % (currency_symbol, self.amount, emailTo)),
```

Comment:
Should close the gettext call first, then interpolate the translated string:

```suggestion
                name=_('Gift certificate: %s%s for %s') % (currency_symbol, self.amount, emailTo),
```